src/components/GameItem.js

```diff
@@ -4,23 +4,43 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false
+    };
+  }
+
+  clickedItem = () => {
+    this.setState({
+      clicked: true,
+    });
+    this.props.type === 'litter' && this.props.clickedItemClickCallback(this.props.index);
```

Comment:
Additionally, because you only check if the item is `litter`... Not a huge thing, but just a tiny bug: This will mean that any time we click on a piece of litter, the points will increase! That means I can click on the same piece of litter a bunch of times and get a bunch of points ;-)